### PR TITLE
Add postmark

### PR DESCRIPTION
**Pull request by aej**
Resolves https://github.com/pentacent/keila/issues/176

### CODE REVIEW

**Review by wmnnd — CHANGES_REQUESTED** [review]
Looking good, except for one tiny typo in the Readme file!

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ With Keila you can easily send out newsletter campaigns and create sign-up
 forms.
 
 For smaller newsletters, you can use your own email inbox to send out campaigns.
-For larger newsletter projects, AWS SES, Sendgrid, and Mailgun are supported in addition
+For larger newsletter projects, AWS SES, Sendgrid, Mailgun and Postmark are supported in addition
```
Comment (aej) [inline README.md:19]:
```suggestion
For larger newsletter projects, AWS SES, Sendgrid, Mailgun, and Postmark are supported in addition
```

[the remark this answers is not in the document]
Comment (aej) [reply README.md]: Done

**Comment by wmnnd** [comment]
Thank you for contributing to Keila :partying_face:

**Comment by aej** [comment]
I've just deployed my own instance of Keila and tried out the new Postmark integration end-to-end. I can confirm it all worked as expected 🎉

**Comment by wmnnd** [comment]
Awesome, thanks for sharing! By the way, if you are serious about using Postmark, you might want to consider adding support for Postmark webhooks for bounces and spam complaints. You can see how I implemented this for SES here: https://github.com/pentacent/keila/blob/main/lib/keila_web/controllers/ses_webhook_controller.ex

**Comment by aej** [comment]
> Awesome, thanks for sharing! By the way, if you are serious about using Postmark, you might want to consider adding support for Postmark webhooks for bounces and spam complaints. You can see how I implemented this for SES here: https://github.com/pentacent/keila/blob/main/lib/keila_web/controllers/ses_webhook_controller.ex

Ah yep - I built these out in my own app (which has its own separate Postmark integration) so I think I could add these pretty easily. Right now I dont have a need for webhook handling in Keila, but as soon as I do I'll add it in.

**Pull request merged**